src/lib/Components/Video.tsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types"
+import React from "react"
+import { Dimensions, requireNativeComponent, ViewProperties } from "react-native"
+
+const { width } = Dimensions.get("window")
+
+interface Props extends ViewProperties {
+  uri?: string
+}
+
+export class Video extends React.Component<Props> {
+  static propTypes = {
+    source: PropTypes.shape({
+      uri: PropTypes.string,
+    }),
+    size: PropTypes.shape({
+      width: PropTypes.number,
+      height: PropTypes.number,
+    }),
+  }
+
+  render() {
+    return (
+      <NativeVideo
+        source={{ uri: "http://techslides.com/demos/sample-videos/small.mp4" }}
+        size={{ width, height: 400 }}
```

Comment:
@orta - Since we're using `CGRect` on the native side I'm not sure how to pass in a `width={...}` and `height={...}` as one would more conventionally see in React UI code. Thoughts?